Documentation/Storage-Configuration/Advanced/ceph-osd-mgmt.md

```diff
@@ -190,3 +190,27 @@ If you don't see a new OSD automatically created, restart the operator (by delet
 
 !!! note
     The OSD might have a different ID than the previous OSD that was replaced.
+
+
+## OSD Encryption as Day-2 operation
+
+Existing PVC based OSDs can be migrated to enable or disable encryption as a day-2 operation. To start migration of the existing OSDs to enable or disable encryption: 
+
+1. Update the cephCluster CR `spec.storage.migration.confirmation` to `yes-really-migrate-osds` and set `StorageClassDevicesets.encrypted` setting to `true` or `false` based on whether encryption needs to be enabled or disabled. For example:
+```yaml
+ storage:
+   migration:
+    confirmation: "yes-really-migrate-osds" 
+   storageClassDeviceSets:
+     - name: set1
+       count: 3
+       encrypted: true  # change to true or false based on whether encryption needs to enable or disabled.
+```
+2. The operator will start the migration of OSDs by destroying the OSD and cleaning up the underlying disk. 
+3. Only one OSD is migrated at a time and operator waits for the data to rebalance (pgs to become `active+clean`) before migrating the next OSD. This ensures that there is no data loss. 
+4. Information about the migration status can be found in cephCluster `status.storage.osd.migrationStatus.pending` which shows the total number of OSDs that are pending migration.
+
+Refer [osd-migration](../../../design/ceph/osd-migration.md) design doc for more information. 
+
+!!! note
+    - The migration involves destroying the OSD and cleaning up the underlying disk. Performance of the cluster might be impacted during data rebalancing while OSDs are being migrated.
```

Comment:
This info was already included above, let's just mention performance in the note
```suggestion
    Performance of the cluster might be impacted during data rebalancing while OSDs are being migrated.
```